my_enum.rb

```diff
@@ -0,0 +1,17 @@
+module MyEnumerable
+  def all?
+    each { |element| return false unless yield(element) }
+    true
+  end
+
+  def any?
+    each { |element| return true if yield(element) }
+    false
+  end
```

Comment:
Kindly, check your `any?` method it should have returned the correct output. I have tested the method with [this](https://github.com/microverseinc/curriculum-ruby/blob/main/simple-ruby/create_your_own_enumerable.md#project-requirements) given inputs and the `any?` method does not give the expected output.

<img width="329" alt="image" src="https://user-images.githubusercontent.com/34578631/174385964-08559de1-1d8d-48b0-a5d0-662c82c4bd60.png">

The `any?` method is returning `false` in every situation. Please fix that.